torch/functional.py

```diff
@@ -396,6 +404,10 @@ def einsum(*args):
         >>> torch.einsum('bn,anm,bm->ba', l, A, r)
         tensor([[-0.3430, -5.2405,  0.4494],
                 [ 0.3311,  5.5201, -3.0356]])
+
+        # sublist format
+        >>> torch.einsum(A, [Ellipsis, 0], l, [1, Ellipsis], [Ellipsis]).shape
```

Comment:
Can you use `...` or must it actually be `Ellipsis`?

Reply:
Would it be more interesting to rewrite one of the above operations using the sublist format, so people could see how it looks when written "normally" and how it look when written as a sublist?

Reply:
`...` also works and yes, I'll rewrite an example above.